MapboxNavigationUI/RouteVoiceController.swift

```diff
@@ -242,16 +242,23 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         return text
     }
     
-    func localizeRoadDescription(_ step: RouteStep) -> String {
+    func localizeRoadDescription(_ step: RouteStep, markUpWithSSML: Bool) -> String {
         var road = ""
+        let escapeIfNecessary = {(distance: String) -> String in
+            return markUpWithSSML ? distance.addingXMLEscapes : distance
+        }
         if let name = step.names?.first {
             if let code = step.codes?.first {
-                road = String.localizedStringWithFormat(NSLocalizedString("NAME_AND_REF", value: "%@ (%@)", comment: "Format for speech string; 1 = way name; 2 = way route number"), name, code)
+                if markUpWithSSML {
+                    road = String.localizedStringWithFormat(NSLocalizedString("NAME_AND_REF", value: "<say-as interpret-as=\"address\">%@</say-as> (<say-as interpret-as=\"address\">%@</say-as>)", comment: "Format for speech string; 1 = way name; 2 = way route number"), name.addingXMLEscapes, code.addingXMLEscapes)
```

Comment:
Any string provided in the .strings files for a particular localization will override this value.

1. You need to provide distinct string IDs for distinct strings.
1. Maybe this isn’t the right place for this markup. Are you sure you don’t want to surround `name.addingXMLEscapes` in `<say-as>` _before_ inserting it into the format string?